inst/codec_data/xx_address/README.md

```diff
@@ -0,0 +1,7 @@
+# Crime
+
+[![latest github release for xx_address dpkg](https://img.shields.io/github/v/release/geomarker-io/codec?sort=date&filter=xx_address-*&display_name=tag&label=%5B%E2%98%B0%5D&labelColor=%238CB4C3&color=%23396175)](https://github.com/geomarker-io/codec/releases?q=xx_address&expanded=false)
+
+Census tract-level measures of crime incidents (including property crimes, violent crimes, other crimes, and gunshots) in Hamilton County, Ohio. Tract-level measures are derived from the data packages stored in the [`xx_address` repository](https://github.com/geomarker-io/xx_address). Version 1.0.1 of the `xx_address` CoDEC data resource harmonizes `crime_incidents-v0.1.1` and `shotspotter-v0.1.1`. View the metadata for each of these data packages for more information about their sources. 
```

Comment:
can we hyperlink to the data resources?

e.g., [`crime-incidents-v0.1.1`](https://github.com/geomarker-io/xx_address/releases/tag/v1.0.1)